src/lib/storage/fixed_size_attribute_vector.cpp

```diff
@@ -0,0 +1,39 @@
+#include "fixed_size_attribute_vector.hpp"
+
+#include <vector>
+#include "type_cast.hpp"
+#include "types.hpp"
+
+namespace opossum {
+
+template <typename T>
+FixedSizeAttributeVector<T>::FixedSizeAttributeVector(const size_t size) {
+  _attribute_vector = std::vector<T>(size);
+}
+
+template <typename T>
+ValueID FixedSizeAttributeVector<T>::get(const size_t i) const {
+  return static_cast<ValueID>(_attribute_vector.at(i));
```

Comment:
Since this might be used in hot loops, we would recommend using `DebugAssert` and `[]`